src/components/vote-form.js

```diff
@@ -1,14 +1,121 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-export default function VoteForm() {
-  return <Wrapper>안녕 나는 투표 폼!</Wrapper>;
+import styled from 'styled-components';
+
+function VoteForm() {
+	const [candi, setCandi] = useState([]);
```

Comment:
```suggestion
	const [candidates, setCandidates] = useState([]);
```
1. initial value로 빈 배열 넘겨주신 것 정말정말정말정말정말x100 좋습니다!! 🥰 
2. 변수,함수명에 약어를 사용하지 말아주세요! 알아보기 쉬운 코드가 좋은 코드입니다 :)
3. 배열을 담고있는 변수이름은 복수형으로 써주세요! (ex : profiles, animals, candidates)